driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala

```diff
@@ -122,7 +125,9 @@ object SparktaClusterJob {
   }
 
   def addPluginsAndClasspath(pluginsPath: String, classPath: String): Seq[URI] = {
-    val config = SparktaConfig.getHdfsConfig.get
+    val zk = new ClusterConfigRepositoryComponent(SparktaConfig.mainConfig.get)
+    val hdfsJsonConfig = new String(zk.repository.get(AppConstant.ConfigZkPath,AppConstant.HdfsId).get)
+    val config = ConfigFactory.parseString(hdfsJsonConfig).getConfig(AppConstant.HdfsId)
```

Comment:
I think that you can use the TypeSafeRepositoryComponent, in the near future we must change the SparktaConfig...